packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -97,6 +97,7 @@ video {
 
 /* Base scaffolding taken from Markswatch theme */
 body {
+  background-color: #fff;
```

Comment:
We don't have to add it here. Adding it below will be sufficient.

Reply:
Done (fadf028)